index.bs

```diff
@@ -3421,7 +3421,7 @@ partial dictionary MLOpSupportLimits {
   <tr>
     <td>*output*</td>
     <td>[=/same type as|same as=] {{a}}</td>
-    <td>maximum of {{a}}'s [=MLOperand/rank=] and {{b}}'s [=MLOperand/rank=]</td>
+    <td>[=/any rank|N=]</td>
```

Comment:
Note: Same here and the following, the allowed ranks, not the output rank calculation.